log/value_test.go

```diff
@@ -10,26 +10,26 @@ import (
 
 func TestValueBinding(t *testing.T) {
 	var output []interface{}
+
 	logger := log.Logger(log.LoggerFunc(func(keyvals ...interface{}) error {
 		output = keyvals
 		return nil
 	}))
-	logger = log.With(logger, "ts", log.Timestamp, "caller", log.Caller)
+	logger = log.With(logger, "ts", log.DefaultTimestamp, "caller", log.DefaultCaller)
 
-	time.Sleep(50 * time.Millisecond)
```

Comment:
This Sleep was present to make sure that the time was the time of the Log call, not the time of the With call. Note that on Windows time is generally reported at maximum granularity of 1ms, and sometimes no better than 15ms. Also on Windows time.Sleep may not have any effect below 15ms.